deployment/helm/kuksa-cloud/templates/cert-manager/issuer.yaml

```diff
@@ -0,0 +1,39 @@
+######################################################################
+# Copyright (c) 2020 Bosch.IO GmbH [and others]
```

Comment:
T highlight that this implementation works only for azure maybe rename file to azure-issuer.yaml